command/agent/node_endpoint_test.go

```diff
@@ -43,13 +43,13 @@ func TestHTTP_NodesList(t *testing.T) {
 		}
 
 		// Check for the index
-		if respW.HeaderMap.Get("X-Nomad-Index") == "" {
+		if respW.Header().Get("X-Nomad-Index") == "" {
```

Comment:
all of these are because `HeaderMap` is deprecated, thought I'd clean up while I was in here